### ISSUE TITLE
Cannot enable cis on Azure 18.04 Pro + FIPS

### ISSUE THREAD

**Issue by cmars** [canonical/ubuntu-advantage-client#1504]
I've made the 'cis' resource available to Pro + FIPS products, but I can't seem to enable it on an instance. I've tried updating to daily with no change in effect.

Tried Azure 18.04 Pro + FIPS.

```
root@cis3:~# ua status
SERVICE       ENTITLED  STATUS    DESCRIPTION
esm-apps      yes       enabled   UA Apps: Extended Security Maintenance (ESM)
esm-infra     yes       enabled   UA Infra: Extended Security Maintenance (ESM)
fips          yes       disabled  NIST-certified FIPS modules
fips-updates  no        —         Uncertified security updates to FIPS modules
livepatch     yes       n/a       Canonical Livepatch service

Enable services with: ua enable <service>
...
```

```
root@cis3:~# ua enable cis
One moment, checking your subscription first
Cannot enable unknown service 'cis'.
Try esm-infra, fips, fips-updates, livepatch.
```

From the machine token, you can see 'cis' is available:
```
         "resourceEntitlements" : [
            {
               "obligations" : {
                  "enableByDefault" : false
               },
               "directives" : {
                  "additionalPackages" : [
                     "usg-cisbenchmark",
                     "usg-common"
                  ],
                  "aptKey" : "E23341B2A1467EDBF07057D6C1997C40EDE22758",
                  "aptURL" : "https://esm.ubuntu.com/cis",
                  "suites" : [
                     "xenial",
                     "bionic"
                  ]
               },
               "entitled" : true,
               "series" : {
                  "xenial" : {
                     "directives" : {
                        "suites" : [
                           "xenial"
                        ]
                     }
                  },
                  "bionic" : {
                     "directives" : {
                        "suites" : [
                           "bionic"
                        ]
                     }
                  }
               },
               "type" : "cis",
               "affordances" : {
...
   "availableResources" : [
      {
         "name" : "cis",
         "available" : true
      },
      {
         "available" : true,
         "name" : "esm-apps"
      },
      {
         "name" : "esm-infra",
         "available" : true
      },
      {
         "name" : "esm-python",
         "available" : false
      },
      {
         "available" : true,
         "name" : "fips"
      },
      {
         "name" : "livepatch",
         "available" : true
      },
      {
         "available" : true,
         "name" : "support"
      }
...
```

**Comment by blackboxsw** [comment]
Apoligies: I may not have been clear in the earlier sync call. UA client considers CIS to be a beta service at the moment. 
default `ua status` won't show it unless you provide `ua status --all`. Beta services cannot be enabled (like cc-eal at the moment) without providing either `ua enable cis --beta`.

Alternatively, to avoid having to add those parameters to ua status and ua enable, adding the following the /etc/ubuntu-advantage/uaclient.conf:
```bash
features:
  allow_beta: true
```

**Comment by blackboxsw** [comment]
We'll leave this issue open until CIS is considered a non-beta service in [ua client release 28.0](https://github.com/canonical/ubuntu-advantage-client/milestone/16).

**Comment by Hooloovoo** [comment]
@blackboxsw perhaps we could also make the error message here more helpful? E.g. "CIS is currently a beta feature, please run ua enable with --beta to enable this"

**Comment by cmars** [comment]
Thanks @blackboxsw! Confirmed CIS is now working on Pro FIPS with --beta.

```
root@testcis:~# ua enable --beta cis
One moment, checking your subscription first
Updating package lists
Installing CIS Audit packages
CIS Audit enabled
root@testcis:~# 
```

**Comment by blackboxsw** [comment]
ubuntu-advantage-tools 27.1 published on 06/17/21 to Ubuntu 16.04 xenial, 18.04 bionic, 20.04 focal, 20.10 groovy, 21.04 hirsute and 21.10 impish .  CIS is no longer considered a `beta` service so the `--beta` parameter is no longer required to enable CIS on LTS releases (16.04, 18.04, 20.04)
